scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -75,6 +75,7 @@ workflow GvsQuickstartIntegration {
     # necessarily the same as the branch name selected in Terra for the integration `GvsQuickstartIntegration` workflow,
     # though in practice likely they are the same.
     if (run_hail_integration) {
+        # This workflow will be the *closest* to AoU test.
```

Comment:
nit: pretty sure I know what you mean but I am struggling to parse this sentence